### DISCUSSION TITLE
Tenho uma assinatura para o Chrome, mas quero para o edeg também, teria como trocar ?

### DISCUSSION THREAD

**Discussion by williamWHC** [aclap-dev/video-downloadhelper#1587] [Help and Q&A] [answered]
### Your question

Tenho uma assinatura para o Chrome, mas gostaria e mudar para o Edge, teria como trocar a minha assinatura atual para o edge ?

### Addon & CoApp version and details

version: 9.1.1.0
target: microsoft
channel: stable
lang: pt-BR
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"key":"d694e44ee0984f60","mismatch":true,"other_browser":"Chrome","this_browser":"Edge"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 Edg/131.0.0.0
{
    "used_history_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 10,
    "successfull_dl": 3,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_1562052838015437"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\willi\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Ressuscita - Ministerio Ipiranga - YouTube.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\willi\\dwhelper\\Ressuscita - Ministerio Ipiranga - YouTube.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=YNZc80t0Arw"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1735497786764
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_2866375695764509"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\willi\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Davido - UNAVAILABLE Official Video ft Musa Keys - YouTube.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\willi\\dwhelper\\Davido - UNAVAILABLE Official Video ft Musa Keys - YouTube.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=OSBan_sH_b8"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1735618632400
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_4021146440801625"
                    },
                    {
                        "download_result": {
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\willi\\dwhelper"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Bruno Mars - The Lazy Song Official Music Video - YouTube.mp4"
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\willi\\dwhelper\\Bruno Mars - The Lazy Song Official Music Video - YouTube.mp4"
                            },
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=fLexgOxsZu0&list=RDfLexgOxsZu0&start_radio=1"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1735618870743
                        }
                    }
                ]
            }
        ]
    },
    "record_download_history": true
}

**Comment by jcfrog** [comment, marked as answer]
Hi,
Sorry we can't swap licenses.
If you purchased less than a month ago we can refund you so that you can purchase Edge premium if you want to.
In that case please send us an email at licenses@downloadhelper.net

**Comment by williamWHC** [comment]
Teria algum desconto ao adquirir uma nova chave ? 

Já que eu já tenho uma hoje.

**Reply by jcfrog** [reply]
no, sorry, there is no discount.
